src/views/home-page/HomePage.vue

```diff
@@ -0,0 +1,116 @@
+<template>
```

Comment:
Trong home page sẽ dùng
<layout>
     <template name="content">
               Rồi đưa toàn bộ nội dung vào đây
     </template>
</layout>

Reply:
Làm tương tự cho các page khác

Thử nghĩ xem tại sao anh lại dùng slot cho master layout nhé